documentation/introduction.tex

```diff
@@ -1,2 +1,11 @@
 \chapter{Introduction}
-Some bla and bla on documentation
+This document describes and documents the software developed in the bachelor project.
+The purpose is to make decisions and structures clear and help developers looking into the source code understanding what they see.
+
+Two applications are developed during the project, a cross-platform front-end mobile application and a web \gls{api} back-end application.
+
+The front-end application written in \gls{typescript} with the \glsi{nativescript} framework and configured to compile to both \textit{Android} and \textit{iOS} operation systems.
+It uses \glsi{ng2} as the application framework and \glsi{apollo:client} for back-end communication.
+
+The back-end application is a \glsi{node} application written in \gls{typescript}.
+It is based on \glsi{express} for web \gls{api} functionality and uses \glsi{graphql} as communication specification.
```

Comment:
mention apollo here i think.?   Maybe : it is an apollo server which is based on express...... (rest of sentence)